encoding/protobuf/error.go

```diff
@@ -59,7 +59,7 @@ func GetErrorCode(err error) yarpcerrors.Code {
 	return yarpcerrors.CodeUnknown
 }
 
-// GetErrorCode returns the error message of the error.
+// GetErrorMessage returns the error message of the error.
```

Comment:
Two thoughts
- Should we also let these methods work with `yarpcerrors.Status` ?
- Should we remove these in favor of just the `yarpcerrors.From*` APIs?

Reply:
I like the second option.